addons/lunch/views/lunch_orders_views.xml

```diff
@@ -224,49 +222,49 @@
                     </div>
                 </div>
                 <div class="o_lunch_wizard">
-                    <div class="row">
-                        <div class="col-2">
-                            <field name="topping_label_1" nolabel="1" attrs="{'invisible': [('available_toppings_1', '=', False)]}" class="fw-bold"/>
+                    <div class="row py-3 py-md-0">
+                        <div class="o_td_label col-3 col-md-2">
+                            <field name="topping_label_1" nolabel="1" attrs="{'invisible': [('available_toppings_1', '=', False)]}" class="o_form_label"/>
                         </div>
-                        <div class="col-10">
-                            <field name="topping_ids_1" attrs="{'invisible': [('available_toppings_1', '=', False)]}" widget="many2many_checkboxes" nolabel="1" domain="[('topping_category', '=', 1), ('supplier_id', '=', supplier_id)]"/>
+                        <div class="col-9 col-md-10">
+                            <field name="topping_ids_1" attrs="{'invisible': [('available_toppings_1', '=', False)]}" widget="many2many_checkboxes" nolabel="1" domain="[('topping_category', '=', 1), ('supplier_id', '=', supplier_id)]" class="o_field_widget o_quick_editable" />
```

Comment:
It tastes like `<group>`, it smells like `<group>` but it's not an actual `<group>`... and I have to admit, I really don't like that. But in this case (cf. having a field as the label for another field), there is no other (that I know of) way to do it :sob: 
with the current tools provided by the framework. Probably something to discuss with them at some point, but out of this task's scope so, let's keep it that way for now :shrug: ...